.changeset/big-days-divide.md

```diff
@@ -0,0 +1,10 @@
+---
+'@backstage/catalog-client': minor
+---
+
+Fixed the return type of the catalog API `getEntityAncestors`, to match the
```

Comment:
Did we break this at some point then? So the error messages weren't being propagated to the user from ancestor entities?

Reply:
Nobody was actually using that particular part of the response. As long as you only read the `entity` part, rather than the stuff that actually described the structure of the graph, you were fine. And that was enough for the processing error panel thing

Reply:
Ah ok, makes sense!